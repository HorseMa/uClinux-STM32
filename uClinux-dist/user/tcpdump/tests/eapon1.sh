#!/bin/sh

uudecode eapon1.puu

echo -n test eapon1...
if (../tcpdump -r eapon1.pcap | diff - eapon1.out)
then
	echo passed.
else
	echo failed.
fi

