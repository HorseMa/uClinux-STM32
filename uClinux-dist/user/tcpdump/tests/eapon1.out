11:17:28.958610 IP 192.168.1.249.netbios-dgm > 192.168.1.255.netbios-dgm: NBT UDP PACKET(138)
11:17:28.958708 IP 192.168.1.249.netbios-dgm > 192.168.1.255.netbios-dgm: NBT UDP PACKET(138)
11:17:28.959360 IP 192.168.1.249.netbios-dgm > 192.168.1.255.netbios-dgm: NBT UDP PACKET(138)
11:17:28.961018 IP 192.168.1.249.netbios-ns > 192.168.1.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:17:29.710899 IP 192.168.1.249.netbios-ns > 192.168.1.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:17:30.461235 IP 192.168.1.249.netbios-ns > 192.168.1.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:17:30.798408 IP 192.168.1.249.netbios-dgm > 192.168.1.255.netbios-dgm: NBT UDP PACKET(138)
11:17:33.464213 IP 192.168.1.249.netbios-ns > 192.168.1.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:17:34.214302 IP 192.168.1.249.netbios-ns > 192.168.1.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:17:34.964688 IP 192.168.1.249.netbios-ns > 192.168.1.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:17:35.473290 arp who-has 192.168.1.1 tell 192.168.1.249
11:17:35.481559 arp reply 192.168.1.1 is-at 00:0d:88:4f:25:91
11:17:35.481577 IP 192.168.1.249.bootpc > 192.168.1.1.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:17:35.622870 EAP code=1 id=0 length=5 
11:17:35.666378 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:17:35.851486 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:17:36.156548 EAP code=1 id=1 length=0 
11:17:36.158698 EAP code=1 id=0 length=5 
11:17:37.766046 EAP code=1 id=0 length=45 
11:17:37.790625 EAP code=1 id=0 length=20 
11:17:37.830669 EAP code=1 id=0 length=76 
11:17:37.848577 EAP code=1 id=0 length=80 
11:17:38.661939 EAP code=1 id=0 length=28 
11:17:38.685352 EAP code=1 id=0 length=4 
11:17:38.686358 EAP code=1 id=3 length=57 
11:17:38.687182 EAP code=1 id=3 length=44 
11:17:39.852392 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:17:46.852719 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:18:02.852731 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:18:08.689384 EAP code=1 id=1 length=0 
11:18:08.696826 EAP code=1 id=0 length=5 
11:18:08.713116 EAP code=1 id=0 length=45 
11:18:08.787664 EAP code=1 id=0 length=20 
11:18:10.344628 EAP code=1 id=0 length=76 
11:18:10.473292 EAP code=1 id=0 length=80 
11:18:11.152435 EAP code=1 id=0 length=28 
11:18:11.251425 EAP code=1 id=0 length=4 
11:18:11.252509 EAP code=1 id=3 length=57 
11:18:11.253336 EAP code=1 id=3 length=44 
11:18:35.856823 arp who-has 169.254.67.194 tell 169.254.67.194
11:18:35.885105 arp who-has 169.254.67.194 tell 169.254.67.194
11:18:36.885304 arp who-has 169.254.67.194 tell 169.254.67.194
11:18:37.907817 IP 169.254.67.194.4299 > 239.255.255.250.1900: UDP, length 133
11:18:37.910524 IP 169.254.67.194 > IGMP.MCAST.NET: igmp v3 report, 1 group record(s)
11:18:37.964030 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:38.691974 IP 169.254.67.194 > IGMP.MCAST.NET: igmp v3 report, 1 group record(s)
11:18:38.714004 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:39.464435 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:39.898479 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:18:40.214836 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:40.909196 IP 169.254.67.194.4299 > 239.255.255.250.1900: UDP, length 133
11:18:40.965632 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:41.254259 EAP code=1 id=1 length=0 
11:18:41.256353 EAP code=1 id=0 length=5 
11:18:41.275901 EAP code=1 id=0 length=45 
11:18:41.388857 EAP code=1 id=0 length=20 
11:18:41.715620 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:42.466013 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:42.963175 EAP code=1 id=0 length=76 
11:18:42.987906 EAP code=1 id=0 length=80 
11:18:43.216408 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:43.675053 EAP code=1 id=0 length=28 
11:18:43.695554 EAP code=1 id=0 length=4 
11:18:43.696547 EAP code=1 id=3 length=57 
11:18:43.697368 EAP code=1 id=3 length=44 
11:18:43.899684 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:18:43.909719 IP 169.254.67.194.4299 > 239.255.255.250.1900: UDP, length 133
11:18:43.967353 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:43.967896 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:44.717196 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:44.718161 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:45.467593 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:45.468557 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:46.217980 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:46.218950 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:46.969929 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:46.970205 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:48.470207 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:49.970986 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:51.471768 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:52.900388 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:18:52.972547 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:53.972751 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:54.972939 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:55.973129 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:18:56.973475 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:57.723686 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:58.474079 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:59.224473 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:18:59.974983 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:19:00.725263 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:19:01.475654 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:19:02.226046 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): REGISTRATION; REQUEST; BROADCAST
11:19:02.976511 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:19:02.976737 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:19:02.977520 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:19:02.979092 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:19:03.728840 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:19:04.479238 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:19:07.482218 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:19:08.232205 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:19:08.982597 IP 169.254.67.194.netbios-ns > 169.254.255.255.netbios-ns: NBT UDP PACKET(137): QUERY; REQUEST; BROADCAST
11:19:09.900631 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from 00:04:23:57:a5:7a, length: 300
11:19:13.696821 EAP code=1 id=1 length=0 
11:19:13.704581 EAP code=1 id=0 length=5 
11:19:13.718221 EAP code=1 id=0 length=45 
11:19:13.734974 EAP code=1 id=0 length=20 
11:19:14.801245 IP 169.254.67.194.netbios-dgm > 169.254.255.255.netbios-dgm: NBT UDP PACKET(138)
11:19:15.293800 EAP code=1 id=0 length=76 
11:19:15.312531 EAP code=1 id=0 length=80 
11:19:15.997763 EAP code=1 id=0 length=28 
11:19:16.022323 EAP code=1 id=0 length=4 
11:19:16.023335 EAP code=1 id=3 length=57 
11:19:16.024149 EAP code=1 id=3 length=44 
